test/parallel_testsuite.py

```diff
@@ -192,6 +218,30 @@ def updateResult(self, result):
     self.buffered_result.updateResult(result)
     result.stopTest(self.test)
     result.core_time += self.test_duration
+    self.log_test_run_for_visualization()
+
+  def log_test_run_for_visualization(self):
+    if os.getenv('EMTEST_VISUALIZE') and (self.test_result != 'skipped' or self.test_duration > 0.2):
+      profiler_logs_path = os.path.join(tempfile.gettempdir(), 'emscripten_toolchain_profiler_logs')
+      os.makedirs(profiler_logs_path, exist_ok=True)
+      profiler_log_file = os.path.join(profiler_logs_path, 'toolchain_profiler.pid_0.json')
```

Comment:
What is the hardcoded `pid_0` here?   Should this be one-file-per-process?

Reply:
No, because the parallel test harness has already joined, and it is only the very last combine printing that does this. (this has then the benefit of not slowing down the actual test run times)